### PR TITLE
Removing SWT authentication handler

### PR DESCRIPTION
**Pull request by mathewc**
Inproc port of https://github.com/Azure/azure-functions-host/pull/10654

### CODE REVIEW

File: test/TestFunctions/TestFunctions.csproj [hunk 1]
```diff
@@ -21,6 +21,8 @@
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Http" Version="3.2.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
     <PackageReference Include="System.Text.Encodings.Web" Version="4.7.2" />
+    <PackageReference Include="System.Net.Http" Version="4.3.4" />
```
Comment (mathewc) [inline test/TestFunctions/TestFunctions.csproj]: Was getting "warning as error" package vulnerability build errors. Copying over references from dev branch project

File: test/WebJobs.Script.Tests.Integration/WebJobs.Script.Tests.Integration.csproj [hunk 2]
```diff
@@ -54,6 +54,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="System.Text.Json" Version="8.0.5" />
```
Comment (mathewc) [inline test/WebJobs.Script.Tests.Integration/WebJobs.Script.Tests.Integration.csproj]: Clean build of this test project was giving me package vulnerability build errors for this package

```
Warning As Error: Package 'System.Text.Json' 8.0.0 has a known high severity vulnerability, https://github.com/advisories/GHSA-8g4q-xg66-9fp4
```

File: test/TestFunctions/TestFunctions.csproj (hunk 1 above)
Comment (jviau) [reply test/TestFunctions/TestFunctions.csproj]: This PR addresses that: #10662

**Review by jviau — APPROVED** [review]

File: test/TestFunctions/TestFunctions.csproj (hunk 1 above)
Comment (mathewc) [reply test/TestFunctions/TestFunctions.csproj]: Rebased on your PR and was able to remove these changes, thanks

**Review by liliankasem — APPROVED** [review]

**Pull request merged**